arch/sim/src/sim/up_usrsock_host.c

```diff
@@ -238,6 +239,7 @@ int usrsock_host_socket(int domain, int type, int protocol)
       return -errno;
     }
 
+  setsockopt(ret, SOL_SOCKET, SO_REUSEADDR, &opt, sizeof(opt));
```

Comment:
why the user doesn't call setsockopt with SO_REUSEADDR by self and then let usrsock forward the request to host side?

Reply:
this is mainly to solve the nuttx abnormal exit and fails to shutdown the socket fd,
the usage of native socket is a little special, not all applications reuse the address in the userspace

Reply:
Ok, it's reasonable. How about add a comment in the source code?

Reply:
done